docker-compose.yml

```diff
@@ -2,12 +2,25 @@ version: "3"
 services:
   redis:
     image: "redis"
+    command: ["redis-server", "--appendonly", "yes"]
+    hostname: redis
+    restart: on-failure
+    volumes:
+      - ~/bridge_data/redis:/data
```

Comment:
What component is responsible for creation `~/bridge_data` on the host system?

Reply:
@akolotov Docker creates all the folder tree automatically while building images with attached drives. Of course, user that build images should have sufficient permissions.